seed/data_importer/tasks.py

```diff
@@ -629,7 +710,10 @@ def _save_raw_data_chunk(chunk, file_pk, progress_key):
                     if key == "bounding_box":  # capture bounding_box GIS field on raw record
                         raw_property.bounding_box = v
                     elif isinstance(v, basestring):
-                        new_chunk[key] = unidecode(v)
+                        try:
+                            new_chunk[key] = unidecode(v)
+                        except Exception:
```

Comment:
I remember having some issues trying to decode the parsed xml at one point and it was throwing on the unidecode... It might have been when I was fooling with the encoding of the xml string. Maybe I fixed that when I added `.decode()` [here](https://github.com/SEED-platform/seed/pull/2144/files/cc520d27a09d7355a02d40c39484183adaf581a0#diff-3132c1cd243c9b15ced1a284b8dbb944R30). Or maybe one of the files I was testing with locally was ASCII encoded. It's probably safe to keep this here right?